docs/pages/ui-programming/add-border-to-text.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Adding a border to text in Expo and React Native apps
+sidebar_title: Adding a border to text
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import SnackInline from '~/components/plugins/SnackInline';
+
+To add a border to a `Text` component in Expo and React Native apps, we will need to use a `View` component.
+
+We will be adding a border to text, similar to the one shown below.
+
+<ImageSpotlight style={{ maxWidth: 276 }} containerStyle={{ marginTop: 0 }} src="/static/images/textborder-example.png" alt="Text rendered with a border" />
+
+## How to add a border to text
+
+To add a border to text, wrap the `Text` component with a `View` component. The `View` component has `style` props that allow us to add a border and customize its appearance.
```

Comment:
```suggestion
To add a border to text, wrap the `Text` component with a `View` component. The `View` component has a `style` prop that allows us to add a border and customize its appearance.
```

Reply:
Some discussion of why we can't style with view rather than on text would be nice.  Something like styles on text are for styling the text itself while borders are about the container and thus the view